examples/acoustic/Acoustic_codegen.py

```diff
@@ -63,16 +64,18 @@ def damp_boundary(damp):
                               dtype=self.dtype, nbpml=nbpml)
         self.src.data[:] = data.get_source()[:, np.newaxis]
 
-    def Forward(self, save=False, cache_blocking=None,
-                auto_tuning=False, cse=True, compiler=None):
-
-        if auto_tuning:
+        if auto_tuning:  # auto tuning with dummy forward operator
             fw = ForwardOperator(self.model, self.src, self.damp, self.data,
                                  time_order=self.t_order, spc_order=self.s_order,
                                  profile=True, save=False, cse=cse, compiler=compiler)
             at = AutoTuner(fw)
             at.auto_tune_blocks(self.s_order + 1, self.s_order * 4 + 2)
```

Comment:
good spot

will fix this tomorrow morning, then merge and finally move straight the next branch